packages/eslint-config-react-native/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-react-native",
+  "description": "STRV's ESLint config for React Native projects",
+  "version": "1.0.0",
```

Comment:
Due to the way `lerna` cuts releases, please set this `version` here to `0.1.0`, and amend the commit introducing this package to contain this text in the commit's body:

```
feat: add @strvcom/eslint-config-react-native

BREAKING CHANGE: Initial release
```

That `BREAKING CHANGE` stanza is important. 😄 Thanks!